.devcontainer/devcontainer.json

```diff
@@ -0,0 +1,17 @@
+{
+  "name": "Node.js & TypeScript",
+  "build": {
+    "dockerfile": "Dockerfile",
+
+    "args": {
+      "VARIANT": "12"
+    },
+    "context": ".."
+  },
+
+  "settings": {},
+
+  "postCreateCommand": "yarn --dev",
+
+  "extensions": ["dbaeumer.vscode-eslint"]
```

Comment:
This is admittedly silly and picky, but I'm a little surprised the formatter didn't remove extra blank lines in this file? 🤔

Reply:
The formatter isn't added yet--that's in PR #346

Reply:
Ah, I see - I keep getting confused by the developer experience, which is a bit self-referential. 💫 
You're working on this PR while connecting to the remote contianer itself - so formatter didn't run, because the pre-commit hook isn't running yet, etc...